packages/fx-core/src/plugins/resource/bot/resources/messages.ts

```diff
@@ -62,6 +62,10 @@ export class Messages {
   public static readonly DeployingBot = "Deploying bot.";
   public static readonly SuccessfullyDeployedBot = "Successfully deployed bot.";
 
+  public static readonly GeneratingArmTemplatesBot = "Generating ARM templates of bot.";
```

Comment:
Duplicate with GENERATE_ARM_TEMPLATES_TITLE

Reply:
The latter is removed.